python/tests/experimental/core/metrics/test_udf_metric.py

```diff
@@ -15,6 +15,9 @@
 logger = getLogger(__name__)
 
 
+# _reset_metric_udfs()
```

Comment:
is this supposed to be here?

Reply:
I was hoping it would reset the UDFs between test files, but pytest doesn't seem to be cooperating. I may have to drop the `_reset_*_udfs()` and use named schemas per test/file now that we [are about to] have that ability.